README.md

```diff
@@ -5,10 +5,11 @@ This plugin allows users to fetch contributor data from GitHub via a slash comma
 ## Installation
 1. Go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-community/releases/latest) and download the latest release for your Mattermost server.
 2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Create a personal access token for your GitHub account [here](https://github.com/settings/tokens). This is required because GitHub has a low rate limit for unauthenticated API requests. You do not need to specify a scope for your token.
+3. Install the GitHub plugin and connect your GitHub account. **Note:** The Community Plugin only works with the ``master`` branch version of the GitHub plugin.
```

Comment:
Is that really the case? I think the minimum requires version of the GitHub plugin is `v2.0,0` https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.0.0.

Reply:
I used various GitHub versions and the Community plugin didn't work with any of them. Joram helped upload the ``master`` version, and then it worked.

Reply:
@hanzei Should I instead open an Issue for this instead of updating the Readme?

Reply:
Yes. Please update the readme to state that v2.0 of the Github plugin (or newer) works. And let's open an issue if that version causes issues for people.

Reply:
@hanzei Sorry for the confusion, my bad. I realized it's the Community plugin itself that had this issue. The master branch of the Community plugin itself has changes that aren't in a release, so me and Joram had to use the master version of the Community plugin instead of the newest release. I've updated the PR description and the Readme now accordingly.